code/src/components/ShoppingSummary.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import { shopping } from 'reducers/shopping'
+import './ShoppingSummary.css'
+
+
+export const ShoppingSummary = () => {
+    const items = useSelector(state => state.shopping.items)
+    const buyMoreItems = items.filter(item => item.needsMore === true)
+
+    const [showShoppingList, setShowShoppingList] = useState(false)
+
+    return (
+        <section className="shoppingSummary">
+            <h4>I have {items.length} items in my fridge</h4>
+            <h2>I need to buy more of {buyMoreItems.length} item
+                {buyMoreItems.length === 1 ? "." : "s."}
```

Comment:
Nice with pluralization! I think it would be a little easier to understand with the entire word in there, but maybe that's just me being slow. The less you have to think when reading code to understand it, the better! Like this:

```js
{buyMoreItems.length === 1 ? "item" : "items"}
```